sdk/python/feast/repo_operations.py

```diff
@@ -111,10 +111,15 @@ def apply_total(repo_config: RepoConfig, repo_path: Path):
     all_to_keep.extend(repo.feature_tables)
     all_to_keep.extend(repo.feature_views)
 
+    entities_to_delete: List[Entity] = []
```

Comment:
Unsure about this `repo_operation.py` file so some help in how to populate this `entities_to_delete` would be nice

Reply:
So basically we just want to clean up any infra for entities that are removed from the feature repo. I think this raises another question: Would we ever want to just apply an operation on a single entity being added/removed, or is it always as part of a feature view or feature table? I don't really see the use case right now. So to me it may make sense to just pass a list of entities alongside "tables_to_delete" and "tables_to_keep" so that we have the necessary information to create/remove infrastructure. Instead of "entities_to_keep". Over time I would like to move entities into feature views/tables so that we dont have to pass it as a separate object. cc @jklegar @tsotnet

Reply:
So more tangibly: Maybe we should just rename these `entities_to_keep` parameters to `entities` and remove all `entities_to_delete` instances.

Reply:
I see. That also makes sense to me 👌 

So we agree that the `entities` would be the ones we keep at any given point right ?

Reply:
Changed

Reply:
Entities would still come and go, but we know which ones are coming and going based on the tables/views.